python/test/test_api.py

```
@@ -164,6 +165,7 @@ def test_memory_allocator_exception(self, server_options):
             ):
                 pass
 
+    @pytest.mark.skip(reason="Skipping test, infer no longer use allocator")
```

we should keep this but refactor - if user requests output memory type gpu and that is not supported by the internal allocator - we would still want to raise an exception during inference

---

yes, we should raise an exception if the output memory type specified on the request is not supported, but currently the bindings does not accept a requested output memory type, so I think we can skip this test for now and add proper testing after adding support for allocating GPU memory.

---

hmm, but if we fail because say cupy is not available? Can we still make sure the right error gets propagated?

---

yes, the "test_unsupported_memory_type" is repurposed for testing moving outputs to unsupported memory type